sources/views/contactsData/usersList.js

```diff
@@ -0,0 +1,67 @@
+import {JetView} from "webix-jet";
+import {contacts} from "models/contacts.js";
+
+export default class UsersListView extends JetView {
+	config() {
+		return {
+			rows: [
+				{
+					view: "list",
+					type:{
+						height:40
+					},
+					localId: "contacts:usersList",
+					template: "<strong>#Name#</strong> (<ins>e-mail</ins>: <em>#Email#</em>) <span class='webix_icon wxi-close btn-right'></span>",
+					select: true,
+					scroll: "y",
+					onClick: {
+						"wxi-close": (e, id) => this.deleteUserInfo(id)
+					}
```

Comment:
here I'd add onAfterSelect event and trigger onChangeUsersListUrl method instead oif calling onChangeUsersListUrl from different places. Will this work?

Reply:
it's working
done